components/brex/actions/list-transaction-for-selected-cash-account/list-transaction-for-selected-cash-account.mjs

```diff
@@ -0,0 +1,21 @@
+import brexApp from "../../brex.app.mjs";
+import common from "./common.mjs";
+
+export default {
+  ...common,
+  name: "List Transactions for Selected Cash Account",
+  description: "Lists all transactions for the specified cash account. [See the docs here](https://developer.brex.com/openapi/transactions_api/#operation/listCashTransactions).",
+  key: "brex-list-transactions-for-selected-cash-account",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    brexApp,
+    cashAccount: {
+      propDefinition: [
+        brexApp,
+        "cashAccount",
+      ],
+    },
```

Comment:
The `cashAccount` prop should be required. If it's not entered, the path for the request becomes `/transactions/cash/undefined`.
```suggestion
    cashAccount: {
      propDefinition: [
        brexApp,
        "cashAccount",
      ],
      optional: false,
    },
```